app/assets/javascripts/common/reports.js

```diff
@@ -390,7 +390,7 @@ DashboardReports = () => {
             {
               label: "Agreed to visit",
               data: Object.values(
-                data.calledWithResultAgreedToVisit
+		capOverduePatientsCallRate(data.calledWithResultAgreedToVisit)
```

Comment:
This caps the individual figures at 100%, but we want to cap the `percentage_called` and `contactable_percentage_called` figures at the step before. I think in the ruby.

We need to use that capped 100% to then calculate this proportional figure.